packages/uicore/src/components/MultiSelectDropDown/MultiSelectDropDown.tsx

```diff
@@ -204,20 +206,26 @@ export function MultiSelectDropDown(props: MultiSelectDropDownProps): React.Reac
     const { handleClick, modifiers } = itemProps
     const isSelected = value && value.findIndex(val => val.value === item.value) > -1
     return (
-      <Checkbox
-        key={item.value.toString()}
-        className={cx(css.menuItem, {
-          [css.active]: isSelected,
-          [css.disabled]: modifiers.disabled || item.disabled
-        })}
-        onClick={e => {
-          if (!modifiers.disabled && !item.disabled) {
-            handleClick(e)
+      <div style={{ marginBottom: '5px' }}>
+        <Checkbox
+          key={item.value.toString()}
+          className={cx(css.menuItem, {
+            [css.active]: isSelected,
+            [css.disabled]: modifiers.disabled || item.disabled
+          })}
+          onClick={e => {
+            if (!modifiers.disabled && !item.disabled) {
+              handleClick(e)
+            }
+          }}
+          checked={isSelected}
+          labelElement={
+            <Layout.Vertical>
+              {item.label} {customItemRender ? customItemRender(item) : null}
+            </Layout.Vertical>
           }
-        }}
-        checked={isSelected}
-        label={item.label}
```

Comment:
to avoid changes to existing dropdowns, let's not remove the `label` prop. just set it conditionally based on wether `itemRenderer` is passed or not.